<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Q-vMS | Quatomotion</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qimplementor/gui/resources/toolbar.qrc">
    <normaloff>:/window_icon/pics/quas.png</normaloff>:/window_icon/pics/quas.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(36, 42, 46);
border-color: rgb(186, 189, 182);
color: rgb(186, 189, 182);
gridline-color: rgb(255, 255, 255);
alternate-background-color: rgb(92, 53, 102);
selection-color: rgb(255, 255, 255);
selection-background-color: rgb(255, 255, 255);
font: 25 10pt &quot;URW Bookman&quot;;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="input_image_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>360</width>
      <height>360</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="start_stop_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 51, 15);
font: 25 14pt &quot;URW Bookman&quot;;</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../qimplementor/gui/resources/toolbar.qrc">:/automotion_icons/pics/Arrow_right.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="add_pre_pro_btn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>159</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 15, 87);</string>
    </property>
    <property name="text">
     <string>+ İşlem Ekle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image_proc_btn">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>410</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../qimplementor/gui/resources/toolbar.qrc">:/automotion_icons/pics/Arrow_right.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>340</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../qimplementor/gui/resources/toolbar.qrc">:/automotion_icons/pics/Arrow_down.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="add_analyse_pro_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>390</y>
      <width>159</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 15, 87);</string>
    </property>
    <property name="text">
     <string>+ İşlem Ekle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="output_graph_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../qimplementor/gui/resources/toolbar.qrc">
      <normaloff>:/Toolbar/pics/plot.png</normaloff>:/Toolbar/pics/plot.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>360</width>
      <height>360</height>
     </size>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="prepro_layout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Canny Filtreleme</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>420</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>161</width>
       <height>62</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="analyse_layout">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Eğri Analizi</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="analyse_layout_2">
        <item>
         <widget class="QPushButton" name="scratch_detection_btn">
          <property name="text">
           <string>Çizik Analizi</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>510</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../qimplementor/gui/resources/toolbar.qrc">:/automotion_icons/pics/Arrow_LEFT.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>510</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../qimplementor/gui/resources/toolbar.qrc">:/automotion_icons/pics/Arrow_LEFT.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="okng_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(164, 0, 0);
font: 25 14pt &quot;URW Bookman&quot;;</string>
    </property>
    <property name="text">
     <string>NG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>500</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../qimplementor/gui/resources/toolbar.qrc">:/automotion_icons/pics/Arrow_LEFT.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="part_name_lbl">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PARÇA ADI: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(24, 16, 16);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Q-vMS Otomasyon Arayüzü&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>861</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>861</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>630</y>
      <width>861</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../qimplementor/gui/resources/toolbar.qrc">:/automotion_icons/pics/Arrow_right.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>650</y>
      <width>202</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 16pt &quot;URW Bookman&quot;;</string>
    </property>
    <property name="text">
     <string>ANALİZ İŞLEMLERİ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>641</y>
      <width>59</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 16pt &quot;URW Bookman&quot;;</string>
    </property>
    <property name="text">
     <string>ÇIKIŞ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>650</y>
      <width>200</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 16pt &quot;URW Bookman&quot;;</string>
    </property>
    <property name="text">
     <string>GÖRÜNTÜ İŞLEME</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>301</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 16pt &quot;URW Bookman&quot;;</string>
    </property>
    <property name="text">
     <string>GİRİŞ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>563</x>
      <y>301</y>
      <width>166</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 16pt &quot;URW Bookman&quot;;</string>
    </property>
    <property name="text">
     <string>ÖN - İŞLEMLER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>300</y>
      <width>166</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 16pt &quot;URW Bookman&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ARA ÇIKTI&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pre_pro_output_btn">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>70</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>360</width>
      <height>360</height>
     </size>
    </property>
   </widget>
   <zorder>label_9</zorder>
   <zorder>label_7</zorder>
   <zorder>label_4</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>input_image_btn</zorder>
   <zorder>start_stop_btn</zorder>
   <zorder>label</zorder>
   <zorder>add_pre_pro_btn</zorder>
   <zorder>image_proc_btn</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>output_graph_btn</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>okng_btn</zorder>
   <zorder>label_12</zorder>
   <zorder>part_name_lbl</zorder>
   <zorder>label_13</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>add_analyse_pro_btn</zorder>
   <zorder>line_3</zorder>
   <zorder>label_14</zorder>
   <zorder>label_15</zorder>
   <zorder>pre_pro_output_btn</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1269</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>172</y>
      <width>153</width>
      <height>162</height>
     </rect>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuYeni">
     <property name="title">
      <string>Yeni</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="actionPRoje"/>
     <addaction name="actionResim"/>
    </widget>
    <widget class="QMenu" name="menuA">
     <property name="title">
      <string>Aç</string>
     </property>
     <addaction name="actionLoadProject"/>
     <addaction name="actionResim_2"/>
    </widget>
    <widget class="QMenu" name="menuKaydet">
     <property name="title">
      <string>Kaydet</string>
     </property>
     <addaction name="menu_toolbar_save_params"/>
     <addaction name="menu_toolbar_save_meas"/>
    </widget>
    <addaction name="menuYeni"/>
    <addaction name="separator"/>
    <addaction name="menuA"/>
    <addaction name="separator"/>
    <addaction name="menuKaydet"/>
    <addaction name="actionFarkl_Kaydet"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Düzenle</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Çalıştır</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuYard_m">
    <property name="title">
     <string>Yardım</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRun"/>
   <addaction name="menuYard_m"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="console">
   <property name="minimumSize">
    <size>
     <width>1227</width>
     <height>73</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(26, 36, 39);</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1221</width>
       <height>111</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1219</width>
        <height>109</height>
       </rect>
      </property>
      <widget class="QLabel" name="infobox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1231</width>
         <height>91</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>1221</width>
         <height>51</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(26, 36, 39);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
       <property name="buddy">
        <cstring>infobox</cstring>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="toolbar_new"/>
   <addaction name="actionLoadProject"/>
   <addaction name="actionRun"/>
   <addaction name="actionSave"/>
   <addaction name="toolbar_plot"/>
  </widget>
  <action name="actionPRoje">
   <property name="text">
    <string>Proje</string>
   </property>
  </action>
  <action name="actionResim">
   <property name="text">
    <string>Resim</string>
   </property>
  </action>
  <action name="actionLoadProject">
   <property name="icon">
    <iconset resource="../qimplementor/gui/resources/toolbar.qrc">
     <normaloff>:/Toolbar/pics/open.png</normaloff>:/Toolbar/pics/open.png</iconset>
   </property>
   <property name="text">
    <string>Proje</string>
   </property>
   <property name="whatsThis">
    <string>Proje Aç</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionResim_2">
   <property name="text">
    <string>Resim</string>
   </property>
  </action>
  <action name="actionFarkl_Kaydet">
   <property name="text">
    <string>Farklı Kaydet</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../.designer/backup/gui/resources/pics/run.png</normaloff>
     <normalon>:/run/pics/run.png</normalon>../../../../../.designer/backup/gui/resources/pics/run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>Çalıştır</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../qimplementor/gui/resources/toolbar.qrc">
     <normaloff>:/Toolbar/pics/save.png</normaloff>:/Toolbar/pics/save.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="whatsThis">
    <string>Kaydet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="menu_toolbar_save_params">
   <property name="text">
    <string>Parametreleri Kaydet</string>
   </property>
  </action>
  <action name="menu_toolbar_save_meas">
   <property name="text">
    <string>Ölçüm Bilgilerini Kaydet</string>
   </property>
  </action>
  <action name="toolbar_new">
   <property name="icon">
    <iconset resource="../qimplementor/gui/resources/toolbar.qrc">
     <normaloff>:/Toolbar/pics/new.png</normaloff>:/Toolbar/pics/new.png</iconset>
   </property>
   <property name="text">
    <string>Yeni</string>
   </property>
  </action>
  <action name="toolbar_plot">
   <property name="icon">
    <iconset resource="../qimplementor/gui/resources/toolbar.qrc">
     <normaloff>:/Toolbar/pics/plot.png</normaloff>:/Toolbar/pics/plot.png</iconset>
   </property>
   <property name="text">
    <string>plot</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources>
  <include location="../qimplementor/gui/resources/toolbar.qrc"/>
 </resources>
 <connections/>
</ui>
